R/primary-coding-suep.R

```diff
@@ -75,7 +75,7 @@ primary_coding_suep_age <- function(trial_data, pid, visitid) {
   new_vars_to_add <- form_to_add_vars %>%
     left_join(select(form_with_needed_vars, -matches(visitid)), by=pid) %>% 
     mutate(
-      ecu_age = calculate_full_years(from = .data$gec_birthdate.date, to = .data$gec_pr_incl_date.date),
```

Comment:
because gec_pr_incl_date was not included in the data for the Lemhöfer project

idea: we could built a workaround, if gec_pr_incl_date is in the data, it will be used, if not, it will use pr_incl_date? What do you think?

Or should we talk to TO that they allways have to export gec_pr_incl_date?